hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-update-jenkins-jobs.yaml

```diff
@@ -4,7 +4,10 @@
     triggers:
         - timed: 'H/15 * * * *'
     builders:
-        - shell: 'curl -fsS https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/update-jobs.sh | /bin/bash -'
```

Comment:
You are correct about that. The annoying thing is that Jenkins doesn't let me view that job or change its config once it has broken.